pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -589,78 +589,126 @@ func TestPreFilterPlugins(t *testing.T) {
 
 }
 
-func TestFilterPlugins(t *testing.T) {
+func TestPlugins(t *testing.T) {
```

Comment:
The function should still be TestFilterPlugins since you are not testing other plugins